python-sdk/src/langwatch/langwatch_api_client/api/default/delete_api_annotations_id.py

```diff
@@ -0,0 +1,155 @@
+from http import HTTPStatus
```

Comment:
what do you think of renaming this to `generated/rest_api_client`? just not to polute the root module naming?

Also, could you add this entire folder to the `.gitattributes` file, with `linguist-generated=true`? It'll mark the files are generated, including in collapsing the files by default in diffs.

Reply:
done